awx/ui_next/src/screens/Credential/shared/CredentialSubForms/GoogleComputeEngineSubForm.jsx

```diff
@@ -91,30 +96,38 @@ const GoogleComputeEngineSubForm = ({ i18n }) => {
           }}
         />
       </FormGroup>
-      <FormField
+      <CredentialPluginField
         id="credential-username"
         label={i18n._(t`Service account email address`)}
         name="inputs.username"
         type="email"
         validate={required(null, i18n)}
         isRequired
-      />
-      <FormField
+      >
+        <TextInput id="credential-username" />
+      </CredentialPluginField>
+      <CredentialPluginField
         id="credential-project"
         label={i18n._(t`Project`)}
         name="inputs.project"
-        type="text"
-      />
+      >
+        <TextInput id="credential-project" />
+      </CredentialPluginField>
       <FormFullWidthLayout>
-        <FormField
+        <CredentialPluginField
           id="credential-sshKeyData"
           label={i18n._(t`RSA private key`)}
           name="inputs.ssh_key_data"
           type="textarea"
-          rows={6}
           validate={required(null, i18n)}
           isRequired
-        />
+        >
+          <TextArea
```

Comment:
Should we use the file picker for ssh key and certificate fields? Given all the customization we'll end up doing for credential plugins I think they'll be simpler.

I'd also argue that they're safer by default: Unless we _also_ add drag-and-drop functionality, by using textareas we are inviting users to copy and paste secrets from their clipboard.

Reply:
talked about this...we will